Examples/r/live_objects/Makefile

```diff
@@ -0,0 +1,17 @@
+TOP        = ../..
```

Comment:
It would be better to add this to the testsuite (assuming there's not already an suitable existing testcase there which just needs enabling for R) - i.e. in `Examples/test-suite` rather than here.  Stuff in `Examples/<language>/` is meant to be a collection of standard things which exist for each language (or in reality each language which somebody's sorted them out for).